lms/templates/wiki/base.html

```diff
@@ -13,6 +13,8 @@
   {% stylesheet 'style-course-vendor' %}
   {% stylesheet 'style-course' %}
 
+	<link rel="stylesheet" type="text/css" href="{% static 'paragon/static/paragon.min.css' %}" />
```

Comment:
@arizzitano and here is the required stylesheet when using paragon static build.